### PR TITLE
TASK-35993: Fix onBoarding mail message

### PR DESCRIPTION
**Pull request by AzmiTouil**

### CODE REVIEW

File: component/web/security/src/main/resources/conf/onBoarding_email_template.html [hunk 1]
```diff
@@ -15,9 +15,9 @@ <h2 style="margin: 0;padding: 0; font-size: 16px;font-family: hevetica regular x
                     <td align="center" style="padding: 25px 0 25px">
                         <table width="90%" cellpadding="0" cellspacing="0" >
                             <tr>
-                                <td align="left" style="line-height: 18px;font-size: 14px;color:#333;font-family:hevetica regular x2;">
-                                    <p style="margin: 0 0 10px 0;line-height: 40px;font-size: 17px">&{onboarding.email.hello} ${USER_DISPLAY_NAME},</p>
-                                    <p style="margin: 0 0 10px 0;line-height:22px">&{onboarding.email.welcome} ${COMPANY_NAME}</p>
+                                <td align="left" style="line-height: 18px;font-size: 14px;color:#333;font-family: 'Helvetica', 'Arial', sans-serif;">
+                                    <p style="margin: 0 0 10px 0;line-height: 40px;font-size: 17px;text-transform: capitalize">&{onboarding.email.hi} ${USER_DISPLAY_NAME},</p>
+                                    <p style="margin: 0 0 10px 0;line-height:22px">&{onboarding.email.welcome} ${COMPANY_NAME}.</p>
```
Comment (azayati) [inline component/web/security/src/main/resources/conf/onBoarding_email_template.html]: We will keep the same labels right now, no modification needed

File: web/portal/src/main/resources/locale/portal/webui_en.properties [hunk 2]
```diff
@@ -1338,11 +1338,11 @@ gatein.forgotPassword.captchaError=Captcha Verification Failed
 ###### Onboarding feature
 onboarding.email.subject=Invitation to join the platform
 onboarding.email.header=Invitation to join the platform
-onboarding.email.hello=Hello
+onboarding.email.hi=Hi
```
Comment (azayati) [inline web/portal/src/main/resources/locale/portal/webui_en.properties]: We will keep the same labels right now, no modification needed

**Review by azayati — CHANGES_REQUESTED** [review]
See review

File: component/web/security/src/main/resources/conf/onBoarding_email_template.html (hunk 1 above)
Comment (AzmiTouil) [reply component/web/security/src/main/resources/conf/onBoarding_email_template.html]: Done!

File: web/portal/src/main/resources/locale/portal/webui_en.properties (hunk 2 above)
Comment (AzmiTouil) [reply web/portal/src/main/resources/locale/portal/webui_en.properties]: Done!

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**